Native debezium-oracle test failure

### Bug description

The debezium-oracle native tests are failing on main branch.

This might be due to the oracle database container and https://github.com/apache/camel-quarkus/issues/7773


Using:
```
openjdk 21.0.8 2025-07-15
OpenJDK Runtime Environment (Red_Hat-21.0.8.0.9-1) (build 21.0.8+9)
OpenJDK 64-Bit Server VM (Red_Hat-21.0.8.0.9-1) (build 21.0.8+9, mixed mode, sharing)
```

<details>
<summary>logs</summary>

```
[INFO] Running org.apache.camel.quarkus.component.debezium.common.it.postgres.DebeziumOracleTest
2025-11-05 09:19:04,212 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 4.15.0 (camel-2) is shutting down (timeout:45s)
2025-11-05 09:19:04,212 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Routes stopped (total:0)
2025-11-05 09:19:04,212 INFO  [io.deb.emb.asy.AsyncEmbeddedEngine] (main) Engine state has changed from 'POLLING_TASKS' to 'STOPPING'
2025-11-05 09:19:04,213 INFO  [io.deb.emb.asy.AsyncEmbeddedEngine] (Camel (camel-2) thread #2 - DebeziumConsumer) Task interrupted while polling.
2025-11-05 09:19:04,213 INFO  [io.deb.con.com.BaseSourceTask] (pool-20-thread-1) Stopping down connector
2025-11-05 09:19:04,299 INFO  [com.git.shy.mys.bin.BinaryLogClient] (blc-keepalive-localhost:32775) threadExecutor is shut down, terminating keepalive thread
2025-11-05 09:19:04,299 INFO  [io.deb.pip.ChangeEventSourceCoordinator] (debezium-mysqlconnector-cq-testing-change-event-source-coordinator) Finished streaming
2025-11-05 09:19:04,300 INFO  [io.deb.con.bin.BinlogStreamingChangeEventSource] (blc-localhost:32775) Stopped reading binlog after 22 events, last recorded offset: {ts_sec=1762330743, file=binlog.000002, pos=1807, server_id=1, event=1}
2025-11-05 09:19:04,300 INFO  [io.deb.pip.sig.SignalProcessor] (pool-20-thread-1) SignalProcessor stopped
2025-11-05 09:19:04,300 INFO  [io.deb.ser.DefaultServiceRegistry] (pool-20-thread-1) Debezium ServiceRegistry stopped.
2025-11-05 09:19:04,300 INFO  [io.deb.uti.Threads] (pool-20-thread-1) Requested thread factory for component JdbcConnection, id = JdbcConnection named = jdbc-connection-close
2025-11-05 09:19:04,300 INFO  [io.deb.uti.Threads] (pool-20-thread-1) Creating thread debezium-jdbcconnection-JdbcConnection-jdbc-connection-close
2025-11-05 09:19:04,301 INFO  [io.deb.jdb.JdbcConnection] (debezium-jdbcconnection-JdbcConnection-jdbc-connection-close) Connection gracefully closed
2025-11-05 09:19:04,301 INFO  [io.deb.uti.Threads] (pool-20-thread-1) Requested thread factory for component JdbcConnection, id = JdbcConnection named = jdbc-connection-close
2025-11-05 09:19:04,301 INFO  [io.deb.uti.Threads] (pool-20-thread-1) Creating thread debezium-jdbcconnection-JdbcConnection-jdbc-connection-close
2025-11-05 09:19:04,302 INFO  [io.deb.jdb.JdbcConnection] (debezium-jdbcconnection-JdbcConnection-jdbc-connection-close) Connection gracefully closed
2025-11-05 09:19:04,302 INFO  [io.deb.emb.asy.AsyncEmbeddedEngine] (main) Stopped task #1 out of 1 tasks (it took 89 ms to stop the task).
2025-11-05 09:19:04,302 INFO  [org.apa.kaf.con.sto.FileOffsetBackingStore] (main) Stopped FileOffsetBackingStore
2025-11-05 09:19:04,302 INFO  [io.deb.emb.asy.AsyncEmbeddedEngine] (Camel (camel-2) thread #2 - DebeziumConsumer) Engine is stopped.
2025-11-05 09:19:04,302 INFO  [io.deb.emb.asy.AsyncEmbeddedEngine] (Camel (camel-2) thread #2 - DebeziumConsumer) Engine state has changed from 'STOPPING' to 'STOPPED'
2025-11-05 09:19:04,303 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 4.15.0 (camel-2) shutdown in 90ms (uptime:12s)
2025-11-05 09:19:04,305 INFO  [io.quarkus] (main) camel-quarkus-integration-test-debezium-grouped stopped in 0.094s
2025-11-05 09:19:04,988 INFO  [org.apa.cam.qua.com.deb.com.it.pos.DebeziumOracleTestResource] (pool-23-thread-1) Starting mirror.gcr.io/gvenzl/oracle-free:23-slim-faststart attempt 1 of 5
2025-11-05 09:19:04,989 INFO  [tc.mir.gcr.io/gvenzl/oracle-free:23-slim-faststart] (pool-23-thread-1) Creating container for image: mirror.gcr.io/gvenzl/oracle-free:23-slim-faststart
2025-11-05 09:19:05,260 INFO  [tc.mir.gcr.io/gvenzl/oracle-free:23-slim-faststart] (pool-23-thread-1) Container mirror.gcr.io/gvenzl/oracle-free:23-slim-faststart is starting: 2fd3d086d15d49acbb786780737e56e535255137b28d76079e8b59400d2fffcf
2025-11-05 09:19:55,537 INFO  [tc.mir.gcr.io/gvenzl/oracle-free:23-slim-faststart] (pool-23-thread-1) Container mirror.gcr.io/gvenzl/oracle-free:23-slim-faststart started in PT50.54846139S
2025-11-05 09:19:55,537 INFO  [tc.mir.gcr.io/gvenzl/oracle-free:23-slim-faststart] (pool-23-thread-1) Container is started (JDBC URL: jdbc:oracle:thin:@localhost:32776/oracle)
2025-11-05 09:19:55,538 INFO  [org.tes.ext.ScriptUtils] (pool-23-thread-1) Executing database script from initOracle.sql
2025-11-05 09:19:55,817 INFO  [org.tes.ext.ScriptUtils] (pool-23-thread-1) Executed database script from initOracle.sql in 279 ms.
2025-11-05 09:19:55,853 INFO  [org.apa.cam.qua.cor.CamelBootstrapRecorder] (main) Apache Camel Quarkus 3.30.0-SNAPSHOT is starting
2025-11-05 09:19:55,853 INFO  [org.apa.cam.mai.MainSupport] (main) Apache Camel (Main) 4.15.0 is starting
2025-11-05 09:19:55,857 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 4.15.0 (camel-3) is starting
2025-11-05 09:19:55,858 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Routes startup (total:0)
2025-11-05 09:19:55,858 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 4.15.0 (camel-3) started in 0ms (build:0ms init:0ms start:0ms boot:37ms)
2025-11-05 09:19:55,860 INFO  [io.quarkus] (main) camel-quarkus-integration-test-debezium-grouped 3.30.0-SNAPSHOT on JVM (powered by Quarkus 3.29.0) started in 0.041s. Listening on: http://localhost:8081
2025-11-05 09:19:55,861 INFO  [io.quarkus] (main) Profile test activated. 
2025-11-05 09:19:55,861 INFO  [io.quarkus] (main) Installed features: [agroal, camel-core, camel-debezium-mongodb, camel-debezium-mysql, camel-debezium-oracle, camel-debezium-postgres, camel-debezium-sqlserver, camel-jackson, camel-mongodb, cdi, jdbc-mssql, jdbc-mysql, jdbc-oracle, jdbc-postgresql, mongodb-client, narayana-jta, resteasy, resteasy-jackson, smallrye-context-propagation, vertx]
2025-11-05 09:20:01,685 INFO  [org.apa.kaf.com.con.AbstractConfig] (executor-thread-1) EmbeddedWorkerConfig values: 
	access.control.allow.methods = 
	access.control.allow.origin = 
	admin.listeners = null
	bootstrap.servers = [localhost:9092]
	client.dns.lookup = use_all_dns_ips
	config.providers = []
	connector.client.config.override.policy = All
	header.converter = class org.apache.kafka.connect.storage.SimpleHeaderConverter
	key.converter = class org.apache.kafka.connect.json.JsonConverter
	listeners = [http://:8083]
	metric.reporters = [org.apache.kafka.common.metrics.JmxReporter]
	metrics.num.samples = 2
	metrics.recording.level = INFO
	metrics.sample.window.ms = 30000
	offset.flush.interval.ms = 60000
	offset.flush.timeout.ms = 5000
	offset.storage.file.filename = /tmp/DebeziumOracleTestResource-store-10972913269060901757
	offset.storage.partitions = 0
	offset.storage.replication.factor = 0
	offset.storage.topic = 
	plugin.discovery = hybrid_warn
	plugin.path = null
	response.http.headers.config = 
	rest.advertised.host.name = null
	rest.advertised.listener = null
	rest.advertised.port = null
	rest.extension.classes = []
	ssl.cipher.suites = null
	ssl.client.auth = none
	ssl.enabled.protocols = [TLSv1.2, TLSv1.3]
	ssl.endpoint.identification.algorithm = https
	ssl.engine.factory.class = null
	ssl.key.password = null
	ssl.keymanager.algorithm = SunX509
	ssl.keystore.certificate.chain = null
	ssl.keystore.key = null
	ssl.keystore.location = null
	ssl.keystore.password = null
	ssl.keystore.type = JKS
	ssl.protocol = TLSv1.3
	ssl.provider = null
	ssl.secure.random.implementation = null
	ssl.trustmanager.algorithm = PKIX
	ssl.truststore.certificates = null
	ssl.truststore.location = null
	ssl.truststore.password = null
	ssl.truststore.type = JKS
	task.shutdown.graceful.timeout.ms = 5000
	topic.creation.enable = true
	topic.tracking.allow.reset = true
	topic.tracking.enable = true
	value.converter = class org.apache.kafka.connect.json.JsonConverter

2025-11-05 09:20:01,686 WARN  [org.apa.kaf.con.run.WorkerConfig] (executor-thread-1) The worker has been configured with one or more internal converter properties ([internal.key.converter, internal.value.converter]). Support for these properties was deprecated in version 2.0 and removed in version 3.0, and specifying them will have no effect. Instead, an instance of the JsonConverter with schemas.enable set to false will be used. For more information, please visit https://kafka.apache.org/documentation/#upgrade and consult the upgrade notesfor the 3.0 release.
2025-11-05 09:20:01,686 INFO  [org.apa.kaf.com.con.AbstractConfig] (executor-thread-1) JsonConverterConfig values: 
	converter.type = key
	decimal.format = BASE64
	replace.null.with.default = true
	schemas.cache.size = 1000
	schemas.enable = false

2025-11-05 09:20:01,686 INFO  [org.apa.kaf.com.con.AbstractConfig] (executor-thread-1) JsonConverterConfig values: 
	converter.type = value
	decimal.format = BASE64
	replace.null.with.default = true
	schemas.cache.size = 1000
	schemas.enable = false

2025-11-05 09:20:01,686 INFO  [io.deb.emb.asy.AsyncEmbeddedEngine] (Camel (camel-3) thread #3 - DebeziumConsumer) Engine state has changed from 'CREATING' to 'INITIALIZING'
2025-11-05 09:20:01,695 INFO  [io.deb.con.CommonConnectorConfig] (Camel (camel-3) thread #3 - DebeziumConsumer) Loading the custom source info struct maker plugin: io.debezium.connector.oracle.OracleSourceInfoStructMaker
2025-11-05 09:20:01,699 INFO  [io.deb.uti.Threads] (Camel (camel-3) thread #3 - DebeziumConsumer) Requested thread factory for component OracleConnector, id = cq-testing named = connection-validation
2025-11-05 09:20:01,699 INFO  [io.deb.uti.Threads] (Camel (camel-3) thread #3 - DebeziumConsumer) Creating thread debezium-oracleconnector-cq-testing-connection-validation
2025-11-05 09:20:01,811 ERROR [io.deb.con.ora.OracleConnector] (debezium-oracleconnector-cq-testing-connection-validation) Failed testing connection for {connector.class=io.debezium.connector.oracle.OracleConnector, log.mining.buffer.drop.on.stop=false, snapshot.locking.mode=shared, transaction.metadata.factory=io.debezium.pipeline.txmetadata.DefaultTransactionMetadataFactory, signal.enabled.channels=source, include.schema.changes=true, log.mining.include.redo.sql=false, record.processing.threads=, errors.retry.delay.initial.ms=300, snapshot.fetch.size=0, snapshot.lock.timeout.ms=10000, offset.commit.policy=io.debezium.engine.spi.OffsetCommitPolicy$PeriodicCommitOffsetPolicy, log.mining.scn.gap.detection.gap.size.min=1000000, database.dbname=FREE, snapshot.tables.order.by.row.count=disabled, offset.storage=org.apache.kafka.connect.storage.FileOffsetBackingStore, log.mining.sleep.time.default.ms=1000, log.mining.batch.size.increment=20000, log.mining.archive.log.only.scn.poll.interval.ms=10000, log.mining.restart.connection=false, legacy.decimal.handling.strategy=false, internal.task.management.timeout.ms=40000, max.batch.size=2048, topic.naming.strategy=io.debezium.schema.SchemaTopicNamingStrategy, snapshot.mode=initial, snapshot.mode.configuration.based.snapshot.data=false, record.processing.shutdown.timeout.ms=1000, openlineage.integration.config.file.path=./openlineage.yml, retriable.restart.connector.wait.ms=10000, snapshot.delay.ms=0, log.mining.strategy=online_catalog, executor.shutdown.timeout.ms=4000, snapshot.mode.configuration.based.snapshot.on.data.error=false, record.processing.order=ORDERED, schema.history.internal.file.filename=/tmp/DebeziumOracleTestResource-history-file-2504506041303854997, tombstones.on.delete=false, offset.storage.file.filename=/tmp/DebeziumOracleTestResource-store-10972913269060901757, decimal.handling.mode=precise, binary.handling.mode=bytes, archive.log.hours=0, value.converter=org.apache.kafka.connect.json.JsonConverter, snapshot.mode.configuration.based.start.stream=false, database.pdb.name=oracle, database.connection.adapter=LogMiner, log.mining.flush.table.name=LOG_MINING_FLUSH, log.mining.buffer.type=memory, signal.poll.interval.ms=5000, errors.retry.delay.max.ms=10000, event.processing.failure.handling.mode=fail, snapshot.max.threads=1, offset.flush.interval.ms=60000, internal.key.converter=org.apache.kafka.connect.json.JsonConverter, log.mining.query.filter.mode=none, schema.name.adjustment.mode=none, log.mining.batch.size.default=20000, offset.storage.replication.factor=0, streaming.delay.ms=0, query.fetch.size=10000, database.query.timeout.ms=600000, log.mining.sleep.time.min.ms=0, unavailable.value.placeholder=__debezium_unavailable_value, poll.interval.ms=500, guardrail.collections.max=0, lob.enabled=false, interval.handling.mode=numeric, heartbeat.topics.prefix=__debezium-heartbeat, log.mining.archive.log.only.mode=false, key.converter=org.apache.kafka.connect.json.JsonConverter, log.mining.sleep.time.max.ms=3000, database.user=c##dbzuser, incremental.snapshot.watermarking.strategy=INSERT_INSERT, heartbeat.interval.ms=0, snapshot.mode.configuration.based.snapshot.on.schema.error=false, schema.history.internal.skip.unparseable.ddl=false, errors.max.retries=-1, database.password=********, name=localhost, internal.value.converter=org.apache.kafka.connect.json.JsonConverter, skipped.operations=t, openlineage.integration.job.description=Debezium change data capture job, log.mining.scn.gap.detection.time.interval.max.ms=20000, extended.headers.enabled=true, max.queue.size=8192, guardrail.collections.limit.action=warn, log.mining.buffer.transaction.events.threshold=0, log.mining.transaction.retention.ms=0, provide.transaction.metadata=false, schema.history.internal.store.only.captured.tables.ddl=false, schema.history.internal.store.only.captured.databases.ddl=false, snapshot.database.errors.max.retries=0, topic.prefix=cq-testing, include.schema.comments=false, offset.storage.partitions=0, sourceinfo.struct.maker=io.debezium.connector.oracle.OracleSourceInfoStructMaker, openlineage.integration.enabled=false, openlogreplicator.port=0, log.mining.batch.size.max=100000, max.queue.size.in.bytes=0, snapshot.mode.configuration.based.snapshot.schema=false, time.precision.mode=adaptive, offset.flush.timeout.ms=5000, database.port=32776, log.mining.sleep.time.increment.ms=200, schema.history.internal=io.debezium.storage.file.history.FileSchemaHistory, record.processing.with.serial.consumer=false, log.mining.session.max.ms=0, database.hostname=localhost, log.mining.batch.size.min=1000, connection.validation.timeout.ms=60000} with user '[database.user,null,[],[],true]': java.lang.RuntimeException: Failed to resolve Oracle database version
	at io.debezium.connector.oracle.OracleConnection.resolveOracleDatabaseVersion(OracleConnection.java:197)
	at io.debezium.connector.oracle.OracleConnection.<init>(OracleConnection.java:115)
	at io.debezium.connector.oracle.OracleConnection.<init>(OracleConnection.java:96)
	at io.debezium.connector.oracle.OracleConnector.lambda$validateConnection$0(OracleConnector.java:86)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

2025-11-05 09:20:01,812 ERROR [io.deb.emb.asy.AsyncEmbeddedEngine] (Camel (camel-3) thread #3 - DebeziumConsumer) Engine has failed with : io.debezium.DebeziumException: Connector configuration is not valid. Unable to connect: Failed to resolve Oracle database version
	at io.debezium.embedded.async.AsyncEmbeddedEngine.validateAndGetConnectorConfig(AsyncEmbeddedEngine.java:803)
	at io.debezium.embedded.async.AsyncEmbeddedEngine.initializeConnector(AsyncEmbeddedEngine.java:372)
	at io.debezium.embedded.async.AsyncEmbeddedEngine.run(AsyncEmbeddedEngine.java:208)
	at org.apache.camel.component.debezium.DebeziumConsumer.lambda$doStart$0(DebeziumConsumer.java:62)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

2025-11-05 09:20:01,812 INFO  [io.deb.emb.asy.AsyncEmbeddedEngine] (Camel (camel-3) thread #3 - DebeziumConsumer) Engine state has changed from 'INITIALIZING' to 'STOPPING'
2025-11-05 09:20:01,812 INFO  [io.deb.emb.asy.AsyncEmbeddedEngine] (Camel (camel-3) thread #3 - DebeziumConsumer) Engine is stopped.
2025-11-05 09:20:01,812 INFO  [io.deb.emb.asy.AsyncEmbeddedEngine] (Camel (camel-3) thread #3 - DebeziumConsumer) Engine state has changed from 'STOPPING' to 'STOPPED'
2025-11-05 09:20:01,812 ERROR [io.deb.emb.asy.AsyncEmbeddedEngine] (Camel (camel-3) thread #3 - DebeziumConsumer) Connector configuration is not valid. Unable to connect: Failed to resolve Oracle database version: io.debezium.DebeziumException: Connector configuration is not valid. Unable to connect: Failed to resolve Oracle database version
	at io.debezium.embedded.async.AsyncEmbeddedEngine.validateAndGetConnectorConfig(AsyncEmbeddedEngine.java:803)
	at io.debezium.embedded.async.AsyncEmbeddedEngine.initializeConnector(AsyncEmbeddedEngine.java:372)
	at io.debezium.embedded.async.AsyncEmbeddedEngine.run(AsyncEmbeddedEngine.java:208)
	at org.apache.camel.component.debezium.DebeziumConsumer.lambda$doStart$0(DebeziumConsumer.java:62)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

Request method:	GET
Request URI:	http://localhost:8081/debezium-oracle/receive
Proxy:			<none>
Request params:	<none>
Query params:	<none>
Form params:	<none>
Path params:	<none>
Headers:		Accept=*/*
Cookies:		<none>
Multiparts:		<none>
Body:			<none>

HTTP/1.1 200 OK
Content-Type: text/plain;charset=UTF-8
content-length: 0
Request method:	GET
Request URI:	http://localhost:8081/debezium-oracle/receive
Proxy:			<none>
Request params:	<none>
Query params:	<none>
Form params:	<none>
Path params:	<none>
Headers:		Accept=*/*
Cookies:		<none>
Multiparts:		<none>
Body:			<none>

HTTP/1.1 200 OK
Content-Type: text/plain;charset=UTF-8
content-length: 0
Request method:	GET
Request URI:	http://localhost:8081/debezium-oracle/receive
Proxy:			<none>
Request params:	<none>
Query params:	<none>
Form params:	<none>
Path params:	<none>
Headers:		Accept=*/*
Cookies:		<none>
Multiparts:		<none>
Body:			<none>

HTTP/1.1 200 OK
Content-Type: text/plain;charset=UTF-8
content-length: 0
Request method:	GET
Request URI:	http://localhost:8081/debezium-oracle/receive
Proxy:			<none>
Request params:	<none>
Query params:	<none>
Form params:	<none>
Path params:	<none>
Headers:		Accept=*/*
Cookies:		<none>
Multiparts:		<none>
Body:			<none>

HTTP/1.1 200 OK
Content-Type: text/plain;charset=UTF-8
content-length: 0
Request method:	GET
Request URI:	http://localhost:8081/debezium-oracle/receive
Proxy:			<none>
Request params:	<none>
Query params:	<none>
Form params:	<none>
Path params:	<none>
Headers:		Accept=*/*
Cookies:		<none>
Multiparts:		<none>
Body:			<none>

HTTP/1.1 200 OK
Content-Type: text/plain;charset=UTF-8
content-length: 0
Request method:	GET
Request URI:	http://localhost:8081/debezium-oracle/receive
Proxy:			<none>
Request params:	<none>
Query params:	<none>
Form params:	<none>
Path params:	<none>
Headers:		Accept=*/*
Cookies:		<none>
Multiparts:		<none>
Body:			<none>

HTTP/1.1 200 OK
Content-Type: text/plain;charset=UTF-8
content-length: 0
Request method:	GET
Request URI:	http://localhost:8081/debezium-oracle/receive
Proxy:			<none>
Request params:	<none>
Query params:	<none>
Form params:	<none>
Path params:	<none>
Headers:		Accept=*/*
Cookies:		<none>
Multiparts:		<none>
Body:			<none>

HTTP/1.1 200 OK
Content-Type: text/plain;charset=UTF-8
content-length: 0
Request method:	GET
Request URI:	http://localhost:8081/debezium-oracle/receive
Proxy:			<none>
Request params:	<none>
Query params:	<none>
Form params:	<none>
Path params:	<none>
Headers:		Accept=*/*
Cookies:		<none>
Multiparts:		<none>
Body:			<none>

HTTP/1.1 200 OK
Content-Type: text/plain;charset=UTF-8
content-length: 0
[ERROR] Tests run: 3, Failures: 2, Errors: 1, Skipped: 0, Time elapsed: 138.1 s <<< FAILURE! -- in org.apache.camel.quarkus.component.debezium.common.it.postgres.DebeziumOracleTest
[ERROR] org.apache.camel.quarkus.component.debezium.common.it.postgres.DebeziumOracleTest.testInsert -- Time elapsed: 66.35 s <<< ERROR!
org.testcontainers.shaded.org.awaitility.core.ConditionTimeoutException: 
Assertion condition 1 expectation failed.
Response body doesn't match expectation.
Expected: a string containing "Best Company_c42c62f9-5c17-4c4a-be44-24dd7159ad18"
  Actual: 
 within 1 minutes.
	at org.testcontainers.shaded.org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
	at org.testcontainers.shaded.org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)
	at org.testcontainers.shaded.org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)
	at org.testcontainers.shaded.org.awaitility.core.ConditionFactory.until(ConditionFactory.java:985)
	at org.testcontainers.shaded.org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:769)
	at org.apache.camel.quarkus.test.support.debezium.AbstractDebeziumTest.testInsert(AbstractDebeziumTest.java:72)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.test.junit.QuarkusTestExtension.runExtensionMethod(QuarkusTestExtension.java:998)
	at io.quarkus.test.junit.QuarkusTestExtension.interceptTestMethod(QuarkusTestExtension.java:846)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	Suppressed: java.lang.RuntimeException: Failed to resolve Oracle database version
		at io.debezium.connector.oracle.OracleConnection.resolveOracleDatabaseVersion(OracleConnection.java:197)
		at io.debezium.connector.oracle.OracleConnection.<init>(OracleConnection.java:115)
		at io.debezium.connector.oracle.OracleConnection.<init>(OracleConnection.java:96)
		at io.debezium.connector.oracle.OracleConnector.lambda$validateConnection$0(OracleConnector.java:86)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Suppressed: io.debezium.DebeziumException: Connector configuration is not valid. Unable to connect: Failed to resolve Oracle database version
		at io.debezium.embedded.async.AsyncEmbeddedEngine.validateAndGetConnectorConfig(AsyncEmbeddedEngine.java:803)
		at io.debezium.embedded.async.AsyncEmbeddedEngine.initializeConnector(AsyncEmbeddedEngine.java:372)
		at io.debezium.embedded.async.AsyncEmbeddedEngine.run(AsyncEmbeddedEngine.java:208)
		at org.apache.camel.component.debezium.DebeziumConsumer.lambda$doStart$0(DebeziumConsumer.java:62)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Suppressed: [CIRCULAR REFERENCE: io.debezium.DebeziumException: Connector configuration is not valid. Unable to connect: Failed to resolve Oracle database version]
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at org.testcontainers.shaded.org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:101)
	at org.testcontainers.shaded.org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:81)
	at org.testcontainers.shaded.org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:103)
	... 10 more

[ERROR] org.apache.camel.quarkus.component.debezium.common.it.postgres.DebeziumOracleTest.testUpdate -- Time elapsed: 10.03 s <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Response body doesn't match expectation.
Expected: a string containing "Even Better Company"
  Actual: 

	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:107)
	at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:238)
	at org.apache.camel.quarkus.test.support.debezium.AbstractDebeziumTest.receiveResponse(AbstractDebeziumTest.java:134)
	at org.apache.camel.quarkus.test.support.debezium.AbstractDebeziumTest.testUpdate(AbstractDebeziumTest.java:98)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.test.junit.QuarkusTestExtension.runExtensionMethod(QuarkusTestExtension.java:998)
	at io.quarkus.test.junit.QuarkusTestExtension.interceptTestMethod(QuarkusTestExtension.java:846)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)

[ERROR] org.apache.camel.quarkus.component.debezium.common.it.postgres.DebeziumOracleTest.testDelete -- Time elapsed: 10.07 s <<< FAILURE!
java.lang.AssertionError: 
1 expectation failed.
Expected status code <204> but was <200>.

	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
	at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
	at io.restassured.specification.ResponseSpecification$statusCode$1.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
	at org.apache.camel.quarkus.test.support.debezium.AbstractDebeziumTest.receiveResponse(AbstractDebeziumTest.java:133)
	at org.apache.camel.quarkus.test.support.debezium.AbstractDebeziumTest.testDelete(AbstractDebeziumTest.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at io.quarkus.test.junit.QuarkusTestExtension.runExtensionMethod(QuarkusTestExtension.java:998)
	at io.quarkus.test.junit.QuarkusTestExtension.interceptTestMethod(QuarkusTestExtension.java:846)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)

```

</details>

--- a/pom.xml
+++ b/pom.xml
@@ -257,7 +257,7 @@
         <nats.container.image>mirror.gcr.io/nats:2.10.18</nats.container.image>
         <opensearch.container.image>mirror.gcr.io/opensearchproject/opensearch:2.9.0</opensearch.container.image>
         <openssh-server.container.image>mirror.gcr.io/linuxserver/openssh-server:version-9.7_p1-r4</openssh-server.container.image>
-        <oracle-debezium.container.image>mirror.gcr.io/gvenzl/oracle-free:23-slim-faststart</oracle-debezium.container.image>
+        <oracle-debezium.container.image>mirror.gcr.io/gvenzl/oracle-free:23.9-slim-faststart</oracle-debezium.container.image>
         <pinecone.container.image>ghcr.io/pinecone-io/pinecone-local:v0.7.0</pinecone.container.image>
         <postgres.container.image>mirror.gcr.io/postgres:15.0</postgres.container.image>
         <postgres-debezium.container.image>mirror.gcr.io/debezium/postgres:15-alpine</postgres-debezium.container.image>

